addons/t9n/models/__init__.py

```diff
@@ -3,3 +3,4 @@
 from . import project
 from . import resource
 from . import translation
+from . import reference
```

Comment:
```suggestion
from . import reference
from . import resource
from . import translation
```
Alphabetical ordering makes it easier to find imports when there are many of them